mlflow/pyfunc/__init__.py

```diff
@@ -592,10 +602,10 @@ def log_model(artifact_path, loader_module=None, data_path=None, code_path=None,
                       path before the model is loaded.
     :param conda_env: Either a dictionary representation of a Conda environment or the path to a
                       Conda environment yaml file. This decribes the environment this model should
-                      be run in. If ``python_model`` is not *None*, the Conda environment must
+                      be run in. If ``python_model`` is not ``None``, the Conda environment must
                       at least specify the dependencies contained in
                       :data:`mlflow.pyfunc.DEFAULT_CONDA_ENV`. If `None`, the default
-                      :data:`mlflow.pyfunc.DEFAULT_CONDA_ENV` environment will be added to the
+                      :data:`mlflow.pyfunc.DEFAULT_CONDA_ENV` environment is added to the
```

Comment:
Thanks for catching this - #1238 also addresses the passive phrasing, so there may be some merge conflicts. Hopefully nothing too major.